gui/src-tauri/tauri.conf.json

```diff
@@ -73,7 +73,7 @@
       "all": true
     },
     "shell": {
-      "open": true
+      "open": "^((https?://\\w+)|(file://)).+"
```

Comment:
shouldnt be needed if you just enter the file path instead of `file://`

Reply:
It *should*, but I kept getting regex errors with file paths because the default regex doesn't match file paths.. so I added the `file://` protocol